ziti/cmd/edge/quickstart.go

```diff
@@ -65,7 +65,7 @@ func NewQuickStartCmd(out io.Writer, errOut io.Writer, context context.Context)
 	cmd := &cobra.Command{
 		Use:   "quickstart",
 		Short: "runs a Controller and Router in quickstart mode",
-		Long: `runs a Controller and Router in quickstart mode. By default, this will create a totally ephemeral network, only valid while running.`,
+		Long:  "runs a Controller and Router as a single process; state is deleted on exit unless --home",
```

Comment:
fix the wording. it reads funny.

Reply:
also use `removed` since it's used in the log. i like the consistent wording

Reply:
also, i kinda liked the old wording better in some ways. i like that it used to drive the fact home that this is really not __intended__ to be used long-term